_posts/2021-11-22-changelog-160.markdown

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #160"
+date: 2021-11-22 13:00:00 +0300
+---
+
+## New Features
+
+* [#7996] Enable [crates local index](https://github.com/intellij-rust/intellij-rust/issues/6463) on stable to provide analysis for dependencies inside `Cargo.toml` manifest. Currently, it offers completions for `crates.io` dependencies' names and versions, inspections for non-existent packages in the index, and automatically updates with cargo crates index updates
```

Comment:
Maybe drop mention of `inspections for non-existent packages` since it's disabled by default for now?